backends/cuda-gen/ceed-cuda-gen-operator-build.cpp

```diff
@@ -28,15 +28,15 @@ static const char *atomicAdd = QUOTE(
 //------------------------------------------------------------------------------
 // Atomic add, for older CUDA
 //------------------------------------------------------------------------------
-__device__ double atomicAdd(double *address, double val) {
+__device__ CeedScalar atomicAdd(CeedScalar *address, CeedScalar val) {
```

Comment:
I think this one should stay `double` or simply be removed with `float`,  `atomicAdd` for `float` is always defined.

Reply:
It's in CUDA-8 and later? We don't test ancient versions and I think it doesn't hurt us to just not support versions less than 8.